index.html

```diff
@@ -100,7 +100,7 @@ <h2 class="visually-hidden">Карточки категорий товаров</
 
     <!--Доставка-->
 
-    <section class="delivery">
+    <section class=".main-container delivery">
```

Comment:
```suggestion
    <section class="main-container delivery">
```
Точка перед названием класса не нужна в разметке.